corehq/apps/accounting/management/commands/cchq_software_plan_bootstrap.py

```diff
@@ -226,30 +216,26 @@ def _ensure_product_and_rate(product_type, edition, dry_run, verbose, apps):
     if edition == SoftwarePlanEdition.ENTERPRISE:
         product.name = "Dimagi Only %s" % product.name
 
-    product_rates = []
-
-    for product_rate in BOOTSTRAP_PRODUCT_RATES[edition]:
-        product_rate = SoftwareProductRate(**product_rate)
-        if dry_run:
-            logger.info("[DRY RUN] Creating Product: %s" % product)
-            logger.info("[DRY RUN] Corresponding product rate of $%d created." % product_rate.monthly_fee)
-        else:
-            try:
-                product = SoftwareProduct.objects.get(name=product.name)
-                if verbose:
-                    logger.info("Product '%s' already exists. Using "
-                                "existing product to add rate."
-                                % product.name)
-            except SoftwareProduct.DoesNotExist:
-                product.save()
-                if verbose:
-                    logger.info("Creating Product: %s" % product)
+    product_rate = SoftwareProductRate(**BOOTSTRAP_PRODUCT_RATES[edition])
+    if dry_run:
+        logger.info("[DRY RUN] Creating Product: %s" % product)
+        logger.info("[DRY RUN] Corresponding product rate of $%d created." % product_rate.monthly_fee)
+    else:
+        try:
+            product = SoftwareProduct.objects.get(name=product.name)
+            if verbose:
+                logger.info("Product '%s' already exists. Using "
+                            "existing product to add rate."
+                            % product.name)
+        except SoftwareProduct.DoesNotExist:
+            product.save()
```

Comment:
nit: This was a bit confusing, as in the `try` statement we are shadowing the `product` variable from above. 
Would maybe be easier to follow if `product = SoftwareProduct(name='%s %s' % (product_type, edition), product_type=product_type)` is here, since then it is explicit which product is getting saved in this line.